man/import.q.sorts.Rd

```diff
@@ -17,69 +18,75 @@
 	}
 	\item{manual.lookup}{
-		A matrix with meaningful short handles (same as in \code{\link{build.q.set}}, \code{\link{import.q.concourse}}) as rownames, and arbitary, unique identifying strings in any of the columns as printed on the q-cards by \code{\link{make.cards}}.
-		Defaults to NULL in which case items are automatically identified by full item hashes, as also used by \code{\link{make.cards}}.
+		A matrix with meaningful short handles (same as in \code{\link{build.q.set}}, \code{\link{import.q.concourse}}) as rownames, and arbitary, unique identifying strings (ID) in any of the columns as printed on the q-cards by \code{\link{make.cards}}.
+		Defaults to \code{NULL} in which case items are automatically identified by full item hashes, as also used by \code{\link{make.cards}}.
 	}
 }
 
 \details{
-Q-sorts in their raw form, as created by participants, have columns as ranks (from, say, -6 to +6) with cards (items) sorted in rows.
-The vertical dimension of q-sorts is \emph{meaningless}.
-For reproduceable research, q-sorts, as all data, should be maintained in their rawest form and any processing should be well-documented.
+	Q sorts in their raw form, as created by participants, have columns as ranks (from, say, \code{-6} to \code{+6}) with cards (items) sorted in rows.
+	The vertical dimension of Q sorts is \emph{meaningless}.
+	For reproduceable research, Q sorts, as all data, should be maintained in their rawest form and any processing should be well-documented.
 
-That is what this function offers, by reading in q-sorts from the form in which they were entered by participants.
+	That is what this function offers, by reading in Q sorts from the form in which they were entered by participants.
 
-Q-sorts are best entered not by typing up the full form of an item, but some unique string printed on the card.
-So as \emph{not} to influence participants, these strings should be meaningless to humans.
-This function offers either a set of arbitrary strings hard entered by a researcher for identification in \code{manual.lookup}, or expects a cryptographic hash of the complete item wording, as conveniently also created by \code{\link{make.cards}}.
-These hash values change whenever \emph{anything} in the full item wordings is changed, and allow a precise idenfication of different versions of an item.
+	Q sorts are best entered not by typing up the full form of an item, but some unique string (ID) printed on the card.
+	So as \emph{not} to influence participants, these strings should be meaningless to humans.
+	This function offers either a set of arbitrary strings hard entered by a researcher for identification in \code{manual.lookup}, or expects a cryptographic hash of the complete item wording, as conveniently also created by \code{\link{make.cards}}.
+	These hash values change whenever \emph{anything} in the full item wordings is changed, and allow a precise idenfication of different versions of an item.
```

Comment:
Please review the two sentences for clarity: Does the function generate the strings? what is a cryptographic hash? (and hash in general?)

Since these are not standard Q method terms, I'd define them somewhere (if they aren't defined yet, and if they appear in any other of the help pages, add a parenthesis linking to the help page where you define them, e.g.:

`The hash values (see \code{\link{make.cards}} for details) change...`